proto/frequenz/api/dispatch/fcr/prequalification.proto

```diff
@@ -10,16 +10,21 @@ syntax = "proto3";
 
 package frequenz.api.dispatch.fcr.prequalification;
 
+message SoCBound {
+  // The upper SoC bound.
+  uint32 upper = 1;
+
+  // The lower SoC bound.
+  uint32 lower = 2;
```

Comment:
Only integer bounds are used?

Reply:
similar to what it was before. i've only put it into a struct.

Reply:
I'd suggest calling the common definition of bounds instead: https://github.com/frequenz-floss/frequenz-api-common/blob/2fefe84c338ab166354601b9fbf4c6ca95cbd710/proto/frequenz/api/common/metrics.proto#L15-L21
This comes from the common repo, and is already used in the dispatch definitions.

Reply:
@tiyash-basu-frequenz bounds or limits? I wonder if we want to distinguish between the limits/bounds the microgrid api provides vs. those that are set by an actor/app for its own usage.